src/main/java/org/mifos/connector/mojaloop/model/PayeeFspFee.java

```diff
@@ -0,0 +1,11 @@
+package org.mifos.connector.mojaloop.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
```

Comment:
DTO looks same as above, it should be reused

Reply:
done